### DISCUSSION TITLE
Random assignment of the participants to the experiment lists?

### DISCUSSION THREAD

**Discussion by zhangchi199133** [jspsych/jsPsych#825] [Q&A] [answered]
Hi! I was wondering whether there were instructions with regard to random assignment of the participants to one of the experiment lists I created? Thank you in advance!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @zhangchi199133 ,

If you've got a separate timeline created for each condition, then you could randomly select one of those timelines like this:

```js
var timeline_condition_a = [...];
var timeline_condition_b = [...];
var timeline_condition_c = [...];

var random_timeline = jsPsych.randomization.sampleWithoutReplacement([timeline_condition_a, timeline_condition_b, timeline_condition_c], 1)[0];
```

**Reply by zhangchi199133** [reply]
Hi @jodeleeuw! Thank you so much for the answer!
